packages/sanity/src/core/form/inputs/PortableText/PortableTextInput.tsx

```diff
@@ -218,7 +218,10 @@ export function PortableTextInput(props: PortableTextInputProps) {
     return items
   }, [members, props])
 
-  const hasFocus = focused || isEditorFocusablePath(focusPath)
+  const hasFocus =
```

Comment:
does this actually mean "has focus within"? If so, would it be better if the variable was named to reflect that?

Reply:
Yeah, that's a clearer name for what it is. 👍 I've pushed a new commit with this change.